src/OrchardCore.Modules/OrchardCore.Queries/Views/Items/QueryBasedContentDeploymentStep.Fields.Edit.cshtml

```diff
@@ -0,0 +1,44 @@
+@model QueryBasedContentDeploymentStepViewModel
+@inject OrchardCore.ContentManagement.Metadata.IContentDefinitionManager ContentDefinitionManager
+@inject OrchardCore.Queries.IQueryManager QueryManager
+
+<style asp-name="codemirror"></style>
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-mode-javascript" at="Foot"></script>
+
+<h5>@T["Content Items"]</h5>
+<span class="hint">@T["The query name which will be used to fetch data."]</span>
+
+<div class="mb-3">
+    <div class="form-check">
+        <input type="checkbox" class="form-check-input" asp-for="ExportAsSetupRecipe" />
+        <label class="form-check-label" asp-for="ExportAsSetupRecipe">@T["Export As Setup recipe"]</label>
+        <span class="hint">@T["Check if the data should be exported as a Setup recipe."]</span>
+    </div>
+    <div class="alert alert-warning">@T["If checked, you will have to manually declare variables for the Content Picker fields, Taxonomy fields, Tags fields and List item ids and replace them where needed in the recipe. Any reference to these content items will be lost."]</div>
```

Comment:
Why is this though? We can otherwise export such relationships.

Reply:
I've added this because I wanted to be consistent with already existing steps.

You can read more here: https://github.com/OrchardCMS/OrchardCore/pull/6372